apps/contact-service/src/__tests__/routes/clubs/member/getClubContacts.test.ts

```diff
@@ -0,0 +1,425 @@
+      Effect.gen(function* (_) {
+        const sql = yield* _(SqlClient.SqlClient)
+
+        user1 = yield* _(createMockedUser('+420733333331'))
+        user2 = yield* _(createMockedUser('+420733333332'))
+
+        yield* _(sql`DELETE FROM club_invitation_link`)
+        yield* _(sql`DELETE FROM club_member`)
+        yield* _(sql`DELETE FROM club`)
+      })
+    )
+  })
+
+  it('Should return club members for users that are in the same club', async () => {
+    await runPromiseInMockedEnvironment(
+      Effect.gen(function* (_) {
+        const app = yield* _(NodeTestingApp)
+        const forClubUuid = generateClubUuid()
+
+        yield* _(
+          app.createClub({
+            body: {
+              club: {
+                clubImageUrl: SOME_URL,
+                name: 'someName',
+                description: Option.some('someDescription'),
+                membersCountLimit: 100,
+                uuid: forClubUuid,
+                validUntil: new Date(),
+              },
+            },
+            query: {
+              adminToken: ADMIN_TOKEN,
+            },
+          })
+        )
+
+        const challengeForUser1 = yield* _(
+          addChallengeForKey(user1.mainKeyPair, user1.authHeaders)({})
+        )
+
+        const challengeForUser2 = yield* _(
+          addChallengeForKey(user2.mainKeyPair, user2.authHeaders)({})
+        )
+
+        const inviteLink1 = yield* _(
+          app.generateClubInviteLinkForAdmin({
+            body: {
+              clubUuid: forClubUuid,
+            },
+            query: {
+              adminToken: ADMIN_TOKEN,
+            },
+          })
+        )
+
+        // user1 joins the club
+        yield* _(
+          app.joinClub({
+            body: {
+              code: inviteLink1.code,
+              contactsImported: false,
+              signedChallenge: challengeForUser1.signedChallenge,
+              publicKey: challengeForUser1.publicKey,
+              notificationToken: Option.some(
+                'someToken' as ExpoNotificationToken
+              ),
+            },
+          })
+        )
+
+        const inviteLink2 = yield* _(
+          app.generateClubInviteLinkForAdmin({
+            body: {
+              clubUuid: forClubUuid,
+            },
+            query: {
+              adminToken: ADMIN_TOKEN,
+            },
+          })
+        )
+
+        // user2 joins the club
+        yield* _(
+          app.joinClub({
+            body: {
+              code: inviteLink2.code,
+              contactsImported: false,
+              signedChallenge: challengeForUser2.signedChallenge,
+              publicKey: challengeForUser2.publicKey,
+              notificationToken: Option.some(
+                'someToken' as ExpoNotificationToken
+              ),
+            },
+          })
+        )
+
+        const secondChallengeForUser1 = yield* _(
+          addChallengeForKey(user1.mainKeyPair, user1.authHeaders)({})
```

Comment:
If you are using the util like this. (with `({})`), let's use `generateAndSignChallenge` since it's more clear then. 

```suggestion
          generateAndSignChallenge(user1.mainKeyPair)
```